src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -786,19 +786,6 @@ public void parseWarnsAboutUnmatchedContentInEntry() throws IOException {
     }
 
     @Test
-    @Ignore("Ignoring because this is an edge case")
```

Comment:
I would like to know, what's happening here. Can the test be adapted to the current behavior and possibly commented, what should happen?

I would need that if someone comes over and wants to replace our parser 😇